Automattic/wp-calypso
# Fix Stepper performance bug - Take II

Depends on https://github.com/Automattic/wp-calypso/pull/97480

This PR is the exact replica of the original PR (https://github.com/Automattic/wp-calypso/pull/97354), but it now depends on #97480.

In another instance of tree-shaking breaking down, Stepper was pulling a ton of JS and CSS that it never runs since https://github.com/Automattic/wp-calypso/pull/67887.

Fixes https://github.com/Automattic/wp-calypso/issues/97337

## Proposed Changes

This moves the `setupErrorLogger` function from a big file full of imports into its own file to help tree shaking.

## Why are these changes being made?
To make Stepper faster. 

## Testing Instructions

1. Go to /setup/onboarding.
2. Make sure dashicons.css is not loaded. 
3. Go to /me/purchases. Make sure the table headings (Site, Product, Status, Payment method) are well aligned. 
4. /start/domains/use-your-domain.
5. Make sure the form is centered.
